ppr-ui/src/App.vue

```diff
@@ -469,6 +456,23 @@ export default class App extends Mixins(AuthMixin) {
         }
       }
       this.setUserInfo(userInfo)
+
+      const accountId = this.getAccountModel?.currentAccount?.id || this.getAccountId
```

Comment:
How do these Id's differ?

Reply:
The way account id is set for different users (staff vs non-staff) is not the same, but I could probably just use the `id` under currentAccount.